aep/general/0134/aep.md.j2

```diff
@@ -211,6 +206,25 @@ message UpdateBookRequest {
 
 {% tab oas %}
 
+```yaml
+paths:
+  /v1/publishers/{publisher}/books:
+    patch:
+      summary: Update a book
+      parameters:
+      - name: allow_missing
+        in: query
+        description: If set to true, and the book is not found, a new book will be created.
+        schema:
+          type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
```

Comment:
I think we want "application/merge-patch+json" here, either instead of or in addition to "application/json".